pkg/resources/manifests/deploy.yaml

```diff
@@ -162,6 +162,12 @@ spec:
             value: "auto"
           - name: INSPEKTOR_GADGET_OPTION_FALLBACK_POD_INFORMER
             value: "true"
+          - name: INSPEKTOR_GADGET_CONTAINERD_SOCKETPATH
+            value: "/run/containerd/containerd.sock"
+          - name: INSPEKTOR_GADGET_CRIO_SOCKETPATH
+            value: "/run/crio/crio.sock"
+          - name: INSPEKTOR_GADGET_DOCKER_SOCKETPATH
+            value: "/run/docker.sock"
```

Comment:
Please add a comment here to please keep these values aligned with the ones in the runtime-client package (where the default values are actually defined)